### PR TITLE
disable cuDNN flash attention head dim 256 on blackwell

### PR DESCRIPTION
**Pull request by Cjkkkk**
not currently supported in cuDNN

### CODE REVIEW

**Comment by Cjkkkk** [comment]
@superbobry Hi, could you help review this, thanks!

File: jax/_src/cudnn/fused_attention_stablehlo.py [hunk 1]
```diff
@@ -385,6 +385,14 @@ def check_compute_capability(capability):
   current = tuple(int(x) for x in d.compute_capability.split("."))
   return current >= target
 
+def is_cuda_compute_capability_equal(capability):
+  if not 'cuda' in xla_bridge.get_backend().platform_version:
+    return False
+  d, *_ = jax.local_devices(backend="gpu")
+  target = tuple(int(x) for x in capability.split("."))
+  current = tuple(int(x) for x in d.compute_capability.split("."))
```
Comment (superbobry) [inline jax/_src/cudnn/fused_attention_stablehlo.py:393]: Not 100% sure, but can't it be 9.0a?

**Review by superbobry — APPROVED** [review]

File: jax/_src/cudnn/fused_attention_stablehlo.py (hunk 1 above)
Comment (Cjkkkk) [reply jax/_src/cudnn/fused_attention_stablehlo.py:393]: I think XLA only returns 9.0 for hopper compute capability, 9.0a is only used for PTX compatibility.
